src/main/java/com/github/tomakehurst/wiremock/http/FormParameter.java

```diff
@@ -29,4 +32,28 @@ public FormParameter(
   public static FormParameter absent(String key) {
     return new FormParameter(key, Collections.emptyList());
   }
+
+  @JsonIgnore
+  @Override
+  public boolean isPresent() {
+    return super.isPresent();
+  }
+
+  @JsonProperty
+  @Override
+  public String key() {
+    return super.key();
+  }
+
+  @JsonProperty
+  @Override
+  public List<String> values() {
+    return super.values();
+  }
+
```

Comment:
Should exposing those APIs to JSON be considered another new feature in th echangelog?

Reply:
Sorry, I am not sure what you meant by release logs? Is it the PR title?
I have added a point in the description why this change is necessary.
The absence of this is actually a bug. The near-miss response deserialization is not possible without the annotation. I can try to reproduce the same and put out another PR just fixing this aspect before we merge this PR.

Reply:
@oleg-nenashev 
I checked it a bit more. Looks like those JSON changes are no longer needed for the tests to pass.
Something must have changed after I had raised the PR. In any case I removed these changes.